deployments/compose/jupyter/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyter/pyspark-notebook:notebook-6.5.4
+FROM jupyter/pyspark-notebook:spark-3.3.2
```

Comment:
I don't understand the tags format, even after reading [this](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-pyspark-notebook).  What's the difference between a `notebook` tag and a `spark` tag?

As a notebook user I probably care more about what notebook version I'm getting than what Spark version it includes.  So I would prefer to have a `notebook-x.y.z` tag here and a comment that it includes Spark 3.3 that works with Delta-IO, than a Spark version that leads to an unknown notebook version.

Reply:
OK, I've switched back to `notebook` tag, pinned to 6.5.3 which includes Spark 3.3.2, and added a note to the Docker file.